ome_zarr/writer.py

```diff
@@ -251,21 +257,40 @@ def write_multiscale(
             if chunks_opt is not None:
                 data = da.array(data).rechunk(chunks=chunks_opt)
                 options["chunks"] = chunks_opt
+            else:
+                options["chunks"] = data.chunks
             da_delayed = da.to_zarr(
                 arr=data,
                 url=group.store,
                 component=str(Path(group.path, str(path))),
-                storage_options=options,
-                compressor=options.get("compressor", zarr.storage.default_compressor),
-                dimension_separator=group._store._dimension_separator,
+                # IF we pass storage_options then dask NEEDS url to be a string
+                storage_options=None,
```

Comment:
This means that any `storage_options` passed in by the user are ignored (except for 'chunks')
and is the most likely blocker on this PR.

Reply:
Is there a way to get the URL from the `group.store`? Then instead of `url=group.store` above we could do `url=path/to/store` and this would allow us to pass in `storage_options`.
Or maybe there just aren't any other `storage_options` we need to support here and it's OK to ignore?

Reply:
If the storage_options only specify arguments for *creating* a store, then they would already be ignored if you're also passing in an existing store.
If you're passing in an existing store, created with various arguments, then you wouldn't want to replace that with a url as I suggested on `Apr 7` above.